src/site/content/en/blog/transformstream/index.md

```diff
@@ -9,8 +9,13 @@ alt: Transformer toy figurine.
 author: thomassteiner
 tags:
   - blog
+  - newly-interoperable
 ---
 
+{% Aside 'celebration' %}
+Woohoo, this web feature is now usable on all browsers.
```

Comment:
```suggestion
This web feature is now available in all three engines!
```

Reply:
Try to avoid "all browsers" as frequently that is not the case, even when it is available in all engines.